src/generators/server-side-rendering/index.ts

```diff
@@ -178,7 +178,12 @@ export default function ssr(
 		function __escape ( html ) {
 			return String( html ).replace( /["'&<>]/g, match => escaped[ match ] );
 		}
-	`.replace(/(\\)?@(\w*)/g, (match: string, escaped: string, name: string) => escaped ? match.slice(1) : generator.alias(name));
+	`.replace(/(\\)?([@#])(\w*)/g, (match: string, escaped: string, sigil: string, name: string) => {
```

Comment:
I see similar (but not identical) code to this in the DOM generator, but I'm not entirely sure what it all does. Have all `#`-sigiled identifiers already been replaced, because we're at the top level?